api/services/sandbox/v1/sandbox.proto

```diff
@@ -16,126 +16,240 @@
 
 syntax = "proto3";
 
-// Sandbox is a v2 runtime extension that allows more complex execution environments for containers.
-// This adds a notion of groups of containers that share same lifecycle and/or resources.
-// A few good fits for sandbox can be:
-// - A "pause" container in k8s, that acts as a parent process for child containers to hold network namespace.
-// - (micro)VMs that launch a VM process and executes containers inside guest OS.
-// containerd in this case remains implementation agnostic and delegates sandbox handling to runtimes.
-// See proposal and discussion here: https://github.com/containerd/containerd/issues/4131
 package containerd.services.sandbox.v1;
 
 import "google/protobuf/any.proto";
 import "google/protobuf/timestamp.proto";
-
-import "github.com/containerd/containerd/api/types/sandbox.proto";
+import "google/protobuf/empty.proto";
 import "github.com/containerd/containerd/api/types/mount.proto";
 
 option go_package = "github.com/containerd/containerd/api/services/sandbox/v1;sandbox";
 
-// Store provides a metadata storage interface for sandboxes. Similarly to `Containers`,
-// sandbox object includes info required to start a new instance, but no runtime state.
-// When running a new sandbox instance, store objects are used as base type to create from.
-service Store {
-	rpc Create(StoreCreateRequest) returns (StoreCreateResponse);
-	rpc Update(StoreUpdateRequest) returns (StoreUpdateResponse);
-	rpc Delete(StoreDeleteRequest) returns (StoreDeleteResponse);
-	rpc List(StoreListRequest) returns (StoreListResponse);
-	rpc Get(StoreGetRequest) returns (StoreGetResponse);
+message Sandbox {
+	string id = 1;
+	google.protobuf.Any spec = 2;
+	repeated Container containers = 3;
+	string task_address = 4;
+	map<string, string> labels  = 5;
+	google.protobuf.Timestamp created_at = 6;
+	google.protobuf.Timestamp updated_at = 7;
+	map<string, google.protobuf.Any> extensions = 8;
```

Comment:
Do we need a field about the resource? For example, network namespace, which was previously passed through the pause container spec.

Reply:
it is all in the spec